contracts/token/EscrowedSyndicateERC20.sol

```diff
@@ -14,6 +14,10 @@ contract EscrowedSyndicateERC20 is ERC20("Synthetic Syndicate Token", "sSYN"), A
    */
   uint256 public constant TOKEN_UID = 0xac3051b8d4f50966afb632468a4f61483ae6a953b74e387a01ef94316d6b7d62;
 
+  constructor(address super_admin) AccessControl(super_admin) {
```

Comment:
Would you call the variable `superAdmin` for consistency and to avoid lint issues?
(Did you ran `pnpm run lint`?)